src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -274,7 +328,7 @@ class AKSCreateContext:
     read_only is set to True when necessary to avoid loop calls, when using the getter function to obtain the value of
     other parameters.
     """
-    def __init__(self, cmd: AzCliCommand, raw_parameters: Dict):
+    def __init__(self, cmd: AzCliCommand, raw_parameters: Dict, mode=DecoratorMode.CREATE):
```

Comment:
the mode is hard to understand. maybe give it a full name like decoratorMode?

Reply:
Good catch, changed in commit [#1897ad4](https://github.com/Azure/azure-cli/pull/19647/commits/1897ad4edc8e7262926591df1f5fdaaafffe8595).